.github/workflows/docker.yml

```diff
@@ -3,10 +3,10 @@ name: Build and Push Docker Image
 on:
   workflow_call:
     inputs:
-      registry:
-        required: false
-        type: string
-        default: "ghcr.io"
+      # registry:
+      #   required: false
+      #   type: string
+      #   default: "ghcr.io"
```

Comment:
Removed this because we'll be using the default DockerHub registry.

Reply:
We need to publish on both github nd docker hub.